google/cloud/documentai_toolbox/wrappers/page.py

```diff
@@ -119,24 +151,60 @@ class _BasePageElement(ABC):
     _page: "Page" = dataclasses.field(repr=False)
 
     @cached_property
-    def text(self):
+    def text(self) -> str:
         """
         Text of the page element.
         """
         return _text_from_layout(
-            layout=self.documentai_object.layout, text=self._page._document_text
+            self.documentai_object.layout, self._page._document_text
         )
 
     @cached_property
-    def hocr_bounding_box(self):
+    def hocr_bounding_box(self) -> Optional[str]:
         """
         hOCR bounding box of the page element.
         """
         return _get_hocr_bounding_box(
-            element_with_layout=self.documentai_object,
-            page_dimension=self._page.documentai_object.dimension,
+            self.documentai_object, self._page.documentai_object.dimension
         )
 
+    @cached_property
+    def _text_segment(self) -> documentai.Document.TextAnchor.TextSegment:
+        """
+        Page element text segment.
```

Comment:
Update this docstring to be more informative.  

The implementation is just a one-liner getting the first text segment, unless this is used in many places why not just inline these attribute accesses?

Reply:
This is specifically brought out into a method so it can be a cached property. This field can be accessed multiple times, and this change resulted in a major speed improvement when exporting hocr.